packages/example/pages/index.tsx

```diff
@@ -436,6 +443,35 @@ const Example = ({ authEnabled }: AppContextProps) => {
                     />
                   </td>
                 </tr>
+                <tr>
+                  <td>
+                    <label htmlFor="showBalance">showCurrentChain</label>
+                  </td>
+                  <td>
+                    <input
+                      checked={showCurrentChainLargeScreen}
+                      id="showCurrentChain"
+                      onChange={(event) => {
+                        setShowCurrentChainLargeScreen(
```

Comment:
Looks like it's hooked the other way around. The first one should be for small screen?

Reply:
Good catch 👍. Thanks 🙏